common/src/main/java/org/wso2/testgrid/common/Status.java

```diff
@@ -42,7 +42,22 @@
     /**
      * Entity execution is running.
      */
-    RUNNING("RUNNING");
+    RUNNING("RUNNING"),
+
+    /**
+     * Entity execution didn't run.
+     */
+    DIDNT_RUN("DIDN'T RUN"),
```

Comment:
Lets use "DID_NOT_RUN" instead of the short form.

Reply:
Fixed in 3b69fff